torch/_inductor/sizevars.py

```diff
@@ -356,6 +356,8 @@ def statically_known_multiple_of(
         """
         Return a bool indicating if it is sound to optimize for the numerator being a multiple of the denominator.
         """
+        if free_unbacked_symbols(numerator) or free_unbacked_symbols(denominator):
```

Comment:
What happens when you run this currently ? I would expect this to just return False if we can't prove it.

 also, for non-view, fresh allocations, we should always be able to guarantee alignment. kind of similar to donated buffers.

Reply:
It just returns False, the problem is that it's very expensive to figure out that we're unable to figure it out